flowauth/frontend/src/RoleScopePicker.jsx

```diff
@@ -2,109 +2,227 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
+/* eslint-disable react/prop-types */
```

Comment:
What happens if we don't disable this?
If it really makes no sense perhaps we can disable in eslint's config globally?

Reply:
It's something we _should_ be doing (according to eslint), but haven't - I'd like to put it in in future, but I think it adds a dependency.